docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -161,9 +163,13 @@ a|Defines a destination folder on a device for a file being uploaded
 !`/sdcard/Pictures`
 
 !===
-
+include::partial$proxy-properties.adoc[]
 |===
 
+include::partial$proxy-meta-tags.adoc[]
+
+IMPORTANT: For device configuration, we need to perform the following steps: install CA certificate(download CA certificate from a browserup repository), configure device proxy, and configure the test app to allow proxy in cases of necessity.
```

Comment:
could you please elaborate on "configure device proxy"?

Reply:
I mean the following steps: 
- go to the device’s Settings and Wifi configuration 
- press on the network you’re going to use
- tap a Proxy field and select Manual  
- add a proxy hostname and a proxy port